x-pack/plugins/apm/public/components/app/ServiceMap/Popover/ServiceMetricList.tsx

```diff
@@ -57,8 +40,8 @@ export function ServiceMetricList({
           defaultMessage: 'Trans. duration (avg.)',
         }
       ),
-      description: isNumber(avgTransactionDuration)
-        ? asDuration(avgTransactionDuration)
+      description: isNumber(transactionKPIs.avgTransactionDuration)
```

Comment:
Any preference between KPIs and "stats"? (I think we called it stats somewhere else)
```suggestion
      description: isNumber(transactionStats.avgTransactionDuration)
```

Reply:
`Stats` is probably better over the KPI acronym. There's always some ambiguity with acronyms in camelcase (`transactionKPIs` vs. `transactionKpis`)